extension/sqlpgq/include/sqlpgq_functions.hpp

```diff
@@ -49,6 +50,20 @@ class SQLPGQFunctions {
 		return functions;
 	}
 
+	static vector<CreateTableFunctionInfo> GetTableFunctions() {
+                vector<CreateTableFunctionInfo> functions;
```

Comment:
Please run `make format-fix` (If you didn't do that already), not sure if that will fix this formatting 'issue' but better be safe than sorry. (I don't care too much about the CI at the moment as we will switch to a different repo and CI in the near future but still